modules/config/URLProcessor.js

```diff
@@ -1,69 +1,94 @@
-/* global config, getConfigParamsFromUrl, interfaceConfig, loggingConfig */
+/* global config, interfaceConfig, loggingConfig */
 
 const logger = require("jitsi-meet-logger").getLogger(__filename);
 
-var configUtils = require('./Util');
-var params = {};
+import { getConfigParamsFromUrl } from '../../react/features/base/config';
 
-if (typeof getConfigParamsFromUrl === 'function') {
-    params = getConfigParamsFromUrl();
-}
+import configUtils from './Util';
 
-var URLProcessor = {
-    setConfigParametersFromUrl: function () {
-        // Convert 'params' to JSON object
-        // We have:
-        // {
-        //   "config.disableAudioLevels": false,
-        //   "config.channelLastN": -1,
-        //   "interfaceConfig.APP_NAME": "Jitsi Meet"
-        // }
-        // We want to have:
-        // {
-        //   "config": {
-        //     "disableAudioLevels": false,
-        //     "channelLastN": -1
-        //   },
-        //   interfaceConfig: {
-        //     APP_NAME: "Jitsi Meet"
-        //   }
-        // }
-        var configJSON = {
+// Parsing config params from URL hash.
+const URL_PARAMS = getConfigParamsFromUrl(window.location);
```

Comment:
We do not want globals in react/ and the contents of this file is to eventually land there. Is there a real need for `URL_PARAMS` to exist as a global and not as a local variable of  `setConfigParametersFromUrl`?